src/models.py

```diff
@@ -159,9 +164,12 @@ def __init__(self, architecture_config, training_config, callbacks_config):
 class PyTorchUNetWeightedStream(BasePyTorchUNet):
     def __init__(self, architecture_config, training_config, callbacks_config):
         super().__init__(architecture_config, training_config, callbacks_config)
-        weighted_loss = partial(multiclass_weighted_cross_entropy,
-                                **get_loss_variables(**architecture_config['weighted_cross_entropy']))
-        loss = partial(mixed_dice_cross_entropy_loss, dice_weight=architecture_config['loss_weights']['dice_mask'],
+        weights_function = partial(get_weights, **architecture_config['weighted_cross_entropy'])
```

Comment:
@jakubczakon I just adapted `PyTorchUNetWeightedStream` to match loss definition in `PyTorchUNetWeighted`. I think @taraspiotr could miss it while refactoring, because of giving up the stream mode